### DISCUSSION TITLE
Transferring nixpkgs-review to nix-community

### DISCUSSION THREAD

**Discussion by fricklerhandwerk** [nix-community/infra#1307] [Q&A] [answered]
https://github.com/Mic92/nixpkgs-review is mentioned in Nixpkgs pull request templates and used by many maintainers. It's well-established for all intents and purposes, and actively maintained. @Mic92 what do you think?

Context: [curating official projects](https://discourse.nixos.org/t/call-for-volunteers-curating-official-projects/45382)

**Comment by Mic92** [comment, marked as answer]
Hi. I like the idea of nix-community and moved many of my projects to me as I saw them fit i.e. buildbot-nix today. However I would like to keep some of them as they are also important for my own recognition as my day job as consultant depends on it. Anecdotally this unfortunately works better if the repository has my name in the URL. For important projects I always try to add more people with commit access based on past contributions.
For nixpkgs-review this is:

- @Artturin 
- @figsoda
and
- @supersandro

I believe that so far their is no problem with lacking maintenance in nixpkgs-review. If this ever changes, we can re-negotiating this again. Also there is nothing in future that would stop people from forking nixpkgs-review into nix-community and changing the URL in nixpkgs to take it from there.
